src/sentry/api/endpoints/organization_pinned_searches.py

```diff
@@ -52,6 +53,17 @@ def put(self, request: Request, organization) -> Response:
             visibility=Visibility.OWNER_PINNED,
             values={"query": result["query"], "sort": result["sort"]},
         )
+
+        GroupSearchView.objects.create_or_update(
+            organizaiton=organization,
+            user_id=request.user.id,
+            values={
+                "name": "Default Search",
+                "query": result["query"],
+                "query_sort": result["sort"],
+                "position": 0,  # 0 position indicates default view
```

Comment:
I think the position should be part of the select criteria, since the unique is `org, user, position`